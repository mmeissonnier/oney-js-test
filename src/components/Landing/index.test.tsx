import React from 'react';
import { render } from '@testing-library/react';
import { Landing } from '.';
import { MemoryRouter } from 'react-router-dom';

describe('components/Landing', () => {
  it('should render correctly', () => {
    const { container } = render(
      <MemoryRouter>
        <Landing />
      </MemoryRouter>
    );
    expect(container).toMatchSnapshot();
  });
});
