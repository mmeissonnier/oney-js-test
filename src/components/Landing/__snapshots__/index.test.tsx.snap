// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components/Landing should render correctly 1`] = `
.c0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  -webkit-flex: 1;
  -ms-flex: 1;
  flex: 1;
  margin: 0;
  color: white;
}

.c1 {
  width: auto;
  height: 100px;
}

<div>
  <div
    class="c0"
  >
    <img
      alt="logo"
      class="c1"
      src="https://x-squad.com/images/logo.png"
    />
    <a
      href="/signin"
    >
      Register with my email
    </a>
  </div>
</div>
`;
